test/integration/context-menu-grid.test.js

```diff
@@ -63,7 +59,7 @@ describe('grid in context-menu', () => {
 
       // Close the context menu with ESC
       esc(document.body);
-      await overlayClosed(contextMenu);
```

Comment:
Since there is no closing animation in the `src` version this was causing a timeout.